pkg/apis/openebs.io/v1alpha1/custom_resource.go

```diff
@@ -110,8 +110,14 @@ type CASTemplateSpec struct {
 	// Defaults are a list of default configurations that may be applied
 	// during provisioning of a CAS volume
 	Defaults []Config `json:"defaultConfig"`
+	// TaskNamespace is the namespace where the tasks
+	// are expected to be found
+	TaskNamespace string `json:"taskNamespace"`
```

Comment:
would it be ever possible that the all runtasks don't run in the same namespace?
If yes, then we should probably make namespace updatable by having it in runtask also.

Reply:
@AmitKumarDas Thanks for the explanation. 
I cannot think of any case where we would require it.